templates/vue/src/components/Lightbox/media/ActivityMedia/Question.vue

```diff
@@ -156,6 +174,9 @@ export default {
             if (key === "text") {
               var tempObj = { type: key, answerData: value }
               previousAnswers.push(tempObj)
+            } else if (key === "multipleChoice") {
+              var multiObj = { type: "tasks", answerData: value }
```

Comment:
Why is the type here different then the key?

Reply:
@EldadZZipori because the tasks is the icon for multiple choice, and the type is being set to the icon in Tapestry(code below)

`<template>
  <div class="tapestry-activity">
    <div class="icon"><tapestry-icon :icon="type" /></div>
    <div v-if="type === 'text'" class="text">{{ answerData }}</div>
    <audio v-if="type === 'audio'" controls :src="answerData"></audio>
    <div v-if="type === 'tasks'">{{ answerData }}</div>
  </div>
</template>`